helm_tests/airflow_core/test_worker.py

```diff
@@ -388,6 +388,29 @@ def test_runtime_class_name_values_are_configurable(self):
 
         assert jmespath.search("spec.template.spec.runtimeClassName", docs[0]) == "nvidia"
 
+    def test_livenessprobe_default_command(self):
```

Comment:
Can you rewrite this test to use parametrize?

Reply:
Done!